implementations/rust/ockam/ockam_identity/src/change/create_key.rs

```diff
@@ -2,23 +2,29 @@ use crate::authenticated_storage::AuthenticatedStorage;
 use crate::change::{IdentityChange, IdentityChangeSignature, IdentitySignedChange, SignatureType};
 use crate::change_history::IdentityChangeHistory;
 use crate::IdentityError::InvalidInternalState;
-use crate::{ChangeIdentifier, Identity, IdentityError, IdentityVault, PrivateKeyAttributes};
+use crate::{ChangeIdentifier, Identity, IdentityError, IdentityVault};
 use core::fmt;
-use ockam_core::vault::{KeyId, PublicKey};
+use ockam_core::compat::string::String;
+use ockam_core::vault::{KeyAttributes, KeyId, PublicKey};
 use ockam_core::{Encodable, Result};
 use serde::{Deserialize, Serialize};
 
 /// Key change data creation
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct CreateKeyChangeData {
     prev_change_id: ChangeIdentifier,
-    key_attributes: PrivateKeyAttributes,
+    key_label: String,
```

Comment:
That would probably break backwards compatibility and therefore break Ockam Orchestrator

Reply:
Interesting, that raises 2 questions in my mind:

 1. do we have tests to check the compatibility with Orchestrator? Because they should have been broken with that change
 2. we should probably have a data structure that is purely used to communicate with the Orchestrator. Are we sending the full structure of changes as-is to the Orchestrator? Where is that done?

Reply:
Change history is overly complex, we can break backward compatibility here once but let's do it soon and in one sweep.

Reply:
We use rust sidecar for Identity implementation, so there are no second implementation and no second structure for `Identity`. The problem is that we have existing persistent identities in our system (e.g. Orchestrator has its Identity stored somewhere in its db), which probably won't deserialize because of the changed structure. Unfortunately, there is no backwards compatibility test

Reply:
Can't we create one right now by hardcoding a fully populated encoded Identity in the test?